src/server/server.ts

```diff
@@ -48,6 +53,56 @@ function isOfficeHost(host: string) {
 }
 
 
+let generatedDirectories = {};
+/* Cleans up generated template directories */
+setInterval(() => {
+    let ids = Object.keys(generatedDirectories);
+    for (let id of ids) {
+        let directoryInfo = generatedDirectories[id];
+        let now = (new Date()).getTime();
+        // If generated file is more than an hour old and isn't being read, delete it from both server and map to clean up clutter
+        if (directoryInfo && !directoryInfo.isBeingRead && (now - directoryInfo.timestamp) > 3600000) {
+            delete generatedDirectories[id];
+            fs.unlink(directoryInfo.name, (err) => {
+                if (err) {
+                    ai.trackEvent('File deletion failed', { name: directoryInfo.name });
+                    return;
+                }
+            });
+        }
+    }
+}, 3600000);
```

Comment:
Would be good to have this constant somewhere at the top of the file.  And to clarify what it means (is it 60 seconds * 60 minutes * ...)?